### ISSUE TITLE
Checkboxes are not grouped together in the get support files modal

### ISSUE THREAD

**Issue by ChiuchiuSorin** [xwikisas/application-admintools#162]
Grouping related radio buttons and checkboxes enables assistive technology to convey the group relationship to users (e.g., "group of 5 radio buttons") : [IBM 1.3.1 Info and relationships](https://www.ibm.com/able/requirements/requirements/#1_3_1)

Moreover, logically related input elements should be grouped using a fieldset, this way the relationship may then provided to the user by user agents.

### Who does this affect?

People using a screen reader, including blind, low vision, and neurodivergent people.
